confluent_kafka/deserializing_consumer.py

```diff
@@ -131,9 +131,9 @@ def poll(self, timeout=-1):
         if self._value_deserializer is not None:
             try:
                 value = self._value_deserializer(value, ctx)
-            except SerializationError as se:
+            except Exception as se:
                 raise ConsumeError(KafkaError._VALUE_DESERIALIZATION,
-                                   reason=se.message,
+                                   reason=str(se),
```

Comment:
is there no way to embed the actual exception, not just the string representation of it? (that looses information)